ansible/configs/ocp4-cluster/default_vars_osp.yml

```diff
@@ -185,3 +188,6 @@ master_instance_type: 4c16g30d
 
 # Worker Instance Type
 worker_instance_type: 4c16g30d
+
+# Compute Flavor
+compute_flavor: 4c16g30d
```

Comment:
I'm not sure I get it. What's the difference between compute_flavor and worker_instance_type? Why is worker_instance_type not enough?